playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -439,3 +452,46 @@ def stack_overflow_enricher(alert: PrometheusKubernetesAlert):
             )
         ]
     )
+
+
+class ForeignLogParams(LogEnricherParams):
+    """
+    :var selectors: List of specific label selectors to retrieve logs from
+    """
+
+    selectors: List[str]
```

Comment:
Personally I would rename to 
`label_selectors`
so users dont accidentally put field selectors there without reading the docs